lib/libxdp/xsk.c

```diff
@@ -363,6 +364,25 @@ int xsk_umem__create(struct xsk_umem **umem_ptr, void *umem_area,
 	return err;
 }
 
+int xsk_umem__create(struct xsk_umem **umem_ptr, void *umem_area,
+		     __u64 size, struct xsk_ring_prod *fill,
+		     struct xsk_ring_cons *comp,
+		     const struct xsk_umem_config *usr_config)
+{
+	return __xsk_umem_create(umem_ptr, -1, umem_area, size,
+				 fill, comp, usr_config);
+}
+
+int xsk_umem__create_with_fd(struct xsk_umem **umem_ptr, int fd,
+			     void *umem_area, __u64 size,
+			     struct xsk_ring_prod *fill,
+			     struct xsk_ring_cons *comp,
+			     const struct xsk_umem_config *usr_config)
+{
+	return __xsk_umem_create(umem_ptr, fd, umem_area, size,
+				 fill, comp, usr_config);
+}
```

Comment:
This wrapper seems a bit pointless? Why not just have `xsk_umem__create()` call the `_with_fd()` variant directly?

Reply:
It seemed easier to read this way, but I can change, no problem.

Reply:
done.